var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -44,7 +45,6 @@ class PyLibensemble(PythonPackage):
     variant("mpmath", default=False, description="Install with mpmath")
```

Comment:
Let's create a single `+extras` variant to control these. This matches the pip installation method.

Reply:
Well, the "extras" are to run all the tests, which use various packages. I don't think its likely one person would need all of these things.